content/yaml/building-a-native-ios-app.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Building a native iOS app
+description: Building an iOS app with YAML.
+weight: 4
+---
+
+With `codemagic.yaml`, you can use Codemagic to build, test and publish native iOS apps. You can read more about how to use codemagic.yaml and see the structure of the file [here](../yaml/yaml).
+
+## Building a native iOS app
+
+For building an iOS app, you need to run the following command in the scripts section:
+
+    - xcode-project build-ipa --project "MyXcodeProject.xcodeproj" --scheme "MyScheme"
+
+## Testing an iOS app
+
+The code for testing an iOS app also goes under `scripts`. The relevant code for a native iOS app looks like this:
+
+    xcodebuild \
+    -workspace MyAwesomeApp.xcworkspace \
+    -scheme MyAwesomeApp \
+    -sdk iphonesimulator \
+    -destination 'platform=iOS Simulator,name=iPhone 6,OS=8.1' \
+    test | xcpretty (edited) 
+
+More examples of testing with YAML can be found [here](../yaml/testing).
+
+## Code signing
+
+A full example of iOS code singing with YAML is available [here](../yaml/distribution).
+
+## Publishing
+
+All generated artifacts can be published to external services. The available integrations currently are email, Slack and App Store Connect. Script examples for all of them are available [here](../yaml/distribution/#publishing).
```

Comment:
Need to make sure that we either say app store connect or app store.

Reply:
There is a difference, we only can publish to App Store Connect, and already from there user can distribute it to App Store, Testflight or something else